src/react/angularWrappers/NgPNPCalculator.js

```diff
@@ -0,0 +1,13 @@
+import { react2angular } from 'react2angular';
+import PnpCalculatorContainer from 'react/components/_academics/PnpCalculator/PnpCalculatorContainer';
+
+angular
```

Comment:
@developish - Why are we modifying the PnpCalculator component? Doesn't seem related to Service Alerts or User Authorizations